cmd/common/transaction.go

```diff
@@ -251,10 +251,11 @@ func SignParaTimeTransaction(
 
 // PrintTransaction prints the transaction which can be either signed or unsigned.
 func PrintTransaction(npa *NPASelection, tx interface{}) {
-	var isParaTimeTx bool
+	_, isConsensusTx := tx.(consensusTx.Transaction)
```

Comment:
No need to have this up here now, move it below, e.g. `if _, isConsensusTx := tx.(consensusTx.Transaction); !isConsensusTx && ...`.